tasks/Node/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+//  mode: "development",
+//  devtool: "source-map",
+  mode: "production",
+  devtool: false,
+  target: "node",
+  entry: {
+    AzureDtlCreateCustomImage: "./src/tasks/AzureDtlCreateCustomImage/task.ts",
+    AzureDtlCreateEnvironment: "./src/tasks/AzureDtlCreateEnvironment/task.ts",
+    AzureDtlCreateVM: "./src/tasks/AzureDtlCreateVM/task.ts",
+    AzureDtlDeleteCustomImage: "./src/tasks/AzureDtlDeleteCustomImage/task.ts",
+    AzureDtlDeleteEnvironment: "./src/tasks/AzureDtlDeleteEnvironment/task.ts",
+    AzureDtlDeleteVM: "./src/tasks/AzureDtlDeleteVM/task.ts",
+    AzureDtlUpdateEnvironment: "./src/tasks/AzureDtlUpdateEnvironment/task.ts"
+  },
+  output: {
+    path: __dirname + "/dist/",
+    filename: "tasks/[name]/task.js"
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js"]
```

Comment:
We don't need the .tsx extension, but there isn't any harm in resolving it.

Reply:
I'll leave it. No harm.